LUTECE-2189 : Add a button to display the password on the login page

rzara opened the pull request:
The button is only added if javascript is active. When first loaded, the password is
displayed with bullets.
Displaying the password can help enter it, and thus encourage choosing a stronger one.

rzara committed:
LUTECE-2189 : Add a button to display the password on the login page

The button is only added if javascript is active. When first loaded, the password is
displayed with bullets.
Displaying the password can help enter it, and thus encourage choosing a stronger one.

jonenst commented on webapp/WEB-INF/templates/admin/admin_login.html:
```diff
@@ -49,4 +49,23 @@
   $("#login-page").css('background-image', bgImg );
 });
+$(document).ready(function() {
+	var buttonGroup = $("<span class='input-group-btn'><button type='button' class='btn btn-flat' title='${i18n('portal.admin.admin_login.showPassword')?html}'><span class='fa fa-eye'></span><span class='sr-only'>${i18n('portal.admin.admin_login.showPassword')?html}</span></button></span>");
```
Should we use "message?js_string?html here" ? This way if we add the ' character it won't break ?

rzara replied:
It seems so, yes (for the occurence inside the `title` attibute)

jonenst replied:
why not all 5 occurences ?

rzara replied:
it cannot hurt to do it everywhere, I guess, but only this occurence is susceptible to break if a `'` is unescaped.

rzara committed:
LUTECE-2189 : Factor duplicated text into variables

?html escaping is sufficient because the values are only enclosed in double
quotes, so single quote escapting is not necessary.
